apps/create-fuels-counter-guide/src/app/script/page.tsx

```diff
@@ -39,7 +39,10 @@ export default function ScriptExample() {
       setResult(value.toString());
     } catch (error) {
       console.error(error);
-      toast.error("Error running script.");
+      toast.error(
+        `Error running script. Please make sure your wallet has enough funds.
```

Comment:
Perhaps we should hyperlink to the Top-up Wallet button here, is that possible with the `tost.error` ?

Reply:
Is it worth having a faucet link that will redirect to either the `local` or `testnet` based on env?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2798/commits/e9ca5aefd2f468ff0ed7680ca9061a9bb0761fe8